File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        cnt = 0
        maxi = 0
        for num in nums:
            if num == 1:
                cnt += 1
                maxi = max(maxi, cnt)
            elif num == 0:
                cnt = 0

        return maxi

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int cnt=0;
        for(int num:nums)
        {
            if(num == 1)
            {
                cnt++;
                max = cnt>max? cnt:max;
            }
            else cnt=0;
        }
        return max;
    }
}